pokecli.py

```diff
@@ -74,9 +74,12 @@ def main():
             logger.log('Exiting PokemonGo Bot', 'red')
             finished = True
             report_summary(bot)
-        except NotLoggedInException:
+        except (NotLoggedInException, ServerBusyOrOfflineException):
             logger.log('[x] Error while connecting to the server, please wait %s minutes' % config.reconnecting_timeout, 'red')
             time.sleep(config.reconnecting_timeout * 60)
+        except ServerSideRequestThrottlingException:
+            logger.log('Server is throttling, reconnecting in 30sec')
+            time.sleep(30)
```

Comment:
You're making the same mistake I did, @douglascamata -- it's 30 seconds if you get throttled 15 times in a row.